iiot/readme.md

```diff
@@ -0,0 +1,8 @@
+https://w3id.org/IIoT is a permanent URI 
+
+for the IIoT ontology for Industrial Internet of Things: 
+
+https://ontoit.github.io/IIoT/IIoT.owl
+
+Maintainer: Liviana Tudor
```

Comment:
@dgarijo — You did not request inclusion of Liviana Tudor's email address nor github ID?

Reply:
tudor.liviana@gmail.com - @OntoIT